x-pack/plugins/fleet/common/constants/preconfiguration.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const PRECONFIGURATION_METADATA_INDEX = '.fleet-preconfiguration';
```

Comment:
We are saving our agent policies in a saved object, should we use a saved object for that too?